README.md

```diff
@@ -41,7 +41,7 @@ Then simply open [`http://localhost:3000`](http://localhost:3000) on your web br
 
 (with npm >= 2)
 ```
-$ npm start [-- STAGING_PATH [HTTP_BASE_URI [PORT]]]
+$ npm start [STAGING_PATH [HTTP_BASE_URI [PORT [RESULT_PATH]]]]
```

Comment:
Yes.
`npm start ../specberus/staging/ http://localhost/specberus/staging/ 3030` works for me&hellip;
What am I missing?